File: content-solutions/greedy/height-queue.js
function solution(people) {
  const sorted = [...people].sort((a, b) => b[0] - a[0] || a[1] - b[1]);
  const line = [];
  for (const [h, k] of sorted) line.splice(k, 0, [h, k]);
  return line;
}

File: content-solutions/greedy/height-queue.java
import java.util.*;

class Solution {
    public int[][] solution(int[][] people) {
        int[][] sorted = people.clone();
        Arrays.sort(sorted, (a, b) -> a[0] != b[0] ? Integer.compare(b[0], a[0]) : Integer.compare(a[1], b[1]));
        List<int[]> line = new ArrayList<>();
        for (int[] p : sorted) line.add(p[1], p);
        return line.toArray(new int[0][]);
    }
}
